inc/Ui/Admin.php

```diff
@@ -136,19 +113,27 @@ protected function showMenuItem($item) {
      * @return array list of plugins with their properties
      */
     protected function getPluginList() {
-        global $INFO;
         global $conf;
 
         $pluginlist = plugin_list('admin');
-        $menu = array();
+        $menu = ['admin' => [], 'manager' => [], 'other' => []];
```

Comment:
Just a little style inconsistency: at other places you use ```array()``` instead of ```[]```. A optional change, just let me know if you want to keep it or not.

Reply:
I guess I will prefer `[]`in the future, but I don't think it's a high priority to replace all old `array()` uses.